### PR TITLE
New Exercise: RNA Transcription

### PR DESCRIPTION
**Pull request by ajborla**

### CODE REVIEW

File: exercises/practice/rna-transcription/zcl_rna_transcription.clas.testclasses.abap [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  PRIVATE SECTION.
+    DATA cut TYPE REF TO zcl_rna_transcription.
+    METHODS setup.
+    METHODS:
+      empty_rna_sequence                    FOR TESTING,
+      rna_complement_of_cytosine_is_guanine FOR TESTING,
+      rna_complement_of_guanine_is_cytosine FOR TESTING,
+      rna_complement_of_thymine_is_adenine  FOR TESTING,
+      rna_complement_of_adenine_is_uracil   FOR TESTING,
```
Comment (larshp) [inline exercises/practice/rna-transcription/zcl_rna_transcription.clas.testclasses.abap:14]: shorten the name of these, they are longer than 30 characters

**Comment by larshp** [comment]
seems to work locally, https://github.com/larshp/exercism-test/commit/8f98dab0f36a9951365a30f97d16c4f4fc46a802

I'll open a pull request to update the runner to latest

**Comment by larshp** [comment]
https://github.com/exercism/abap-test-runner/pull/188

File: exercises/practice/rna-transcription/zcl_rna_transcription.clas.testclasses.abap (hunk 1 above)
Comment (ajborla) [reply exercises/practice/rna-transcription/zcl_rna_transcription.clas.testclasses.abap]: I have shortened the unit test names, and this has resolved this issue. Thank you.

**Comment by larshp** [comment]
so, earlier the ABAP code was correct but a false positive

now the ABAP code is incorrect with a correctly failing unit test

**Comment by ajborla** [comment]
I will review code, and resubmit a PR

**Pull request closed without merging**